bridge/api/api.go

```diff
@@ -44,6 +45,7 @@ func New(cfg *bridge.Config) bridge.Bridger {
 	e.GET("/api/health", b.handleHealthcheck)
 	e.GET("/api/messages", b.handleMessages)
 	e.GET("/api/stream", b.handleStream)
+	e.GET("/api/socket", b.handleWebsocket)
```

Comment:
```suggestion
	e.GET("/api/websocket", b.handleWebsocket)
```
I think this is a better name

Reply:
Renamed in df6875d.